charts/pgadmin4/values.yaml

```diff
@@ -130,6 +130,86 @@ serverDefinitions:
   #    SSLMode: "prefer"
   #    MaintenanceDB: "postgres"
 
+oauth2:
+  # Enable/disable OAuth2 authentication
+  enabled: false
+  # The configmap name that holds the OAuth2 client ID and secret
+  clientId: ""
+  # The key in the secret that holds the OAuth2 client secret
+  clientSecret: ""
+
+config_local:
+  # The config_local.py file is used to override the default pgAdmin4 configuration
+  # and is mounted as a ConfigMap in the container.
+  #
+  # Ref: https://www.pgadmin.org/docs/pgadmin4/latest/container_deployment.html#config-local-py
+  #
+  enabled: false
+  # The config_local.py file contents
+  data: |-
+    import os
+    MASTER_PASSWORD_REQUIRED = True
+    AUTHENTICATION_SOURCES = ['oauth2', 'internal']
+    OAUTH2_AUTO_CREATE_USER = True
+    OAUTH2_CONFIG = [
+      {
```

Comment:
Would it be an idea to have a single example here, and others within examples with a reference link towards that location?